GenerateCLP/GenerateCLP.cxx

```diff
@@ -560,6 +560,39 @@ void GenerateDeSerialization( std::ostream & sout,
             | "          deserializedVectorFlaggedArgs.push_back(\"" + flag + "\");"
```

Comment:
When you write 

`Multiple arguments can either be flagged or positional` 

do you mean

 `Argument described using "multiple=true" can either be flagged or positional`